src/renderer/components/spotlight/LayerListItem.js

```diff
@@ -1,23 +1,34 @@
 import React from 'react'
-import { ListItemText, ListItemSecondaryAction, Switch } from '@material-ui/core'
+import { IconButton, ListItemText, ListItemSecondaryAction, Switch, ListItemIcon } from '@material-ui/core'
+import SaveAlt from '@material-ui/icons/SaveAlt'
 import PropTypes from 'prop-types'
 
-const LayerListItem = props => (
-  <div>
-    <ListItemText primary={ props.label } secondary={ props.tags.join(' ') }/>
+const SecondaryAction = p => {
+  if (!p.func) return null
+  return (
     <ListItemSecondaryAction>
-      <Switch
-        edge="end"
-        checked={ props.checked }
-      />
+      <IconButton edge="end" onClick={ p.func }>
+        <SaveAlt />
+      </IconButton>
     </ListItemSecondaryAction>
-  </div>
+  )
+}
+
+const LayerListItem = props => (
+  <React.Fragment>
+    <ListItemIcon>
+      <Switch edge="start" checked={ props.checked } />
+    </ListItemIcon>
+    <ListItemText primary={ props.label } secondary={ props.tags.join(' ') }/>
+    <SecondaryAction func={ props.onSecondaryActionClicked } />
```

Comment:
I would rather propagate layer id in props and use it to push save/export event to ipc/layer, preferably through evented, which ipc/layer already depends upon.
When SecondaryAction should be re-used for other actions, propagating an `action` function could be a way to go.